README.md

```diff
@@ -21,6 +21,7 @@ section:
 | `generate_pyi` | `true` | Whether to generate .pyi output files. Note that these are not generated for the gRPC output. You may want to use mypy-protobuf instead. |
 | `proto_paths` | `["."]` or `["src"]` | An array of paths to search for `.proto` files. Also passed as `--proto_path` arguments to `protoc`. |
 | `output_path` | `"."` or `"src"` | The default output directory. This can be overridden on a per-generator basis for custom generators. |
+| `includes` | `[]` | Additional include directories to pass to `protoc` via `-I` flag. |
```

Comment:
As with my other comment, I'd like to be consistent about using `-I` vs `--proto_path`. I think it would also be worth explicitly stating (either in the table or below it) that `proto_paths` and `includes` _both_ add to the list of places that `protoc` searches for includes, but only files in `proto_paths` will be generated into code